Gemfile.lock

```diff
@@ -483,7 +483,7 @@ GEM
       rspec-mocks (~> 3.5.0)
       rspec-support (~> 3.5.0)
     rspec-support (3.5.0)
-    rubocop (0.47.1)
+    rubocop (0.48.1)
```

Comment:
I like that idea! created an issue for it here: https://github.com/18F/identity-private/issues/1827

for now I created a separate PR for rubocop update since correcting got pretty messy. will revert the rubocop-specific commits from this PR